cdk/domino_cdk/provisioners/eks/eks_nodegroup.py

```diff
@@ -8,6 +8,10 @@
 
 from domino_cdk import config
 
+from ..ami import root_device_mapping
+
+NodeGroup = TypeVar("NodeGroup", bound=config.EKS.NodegroupBase)
```

Comment:
Might be worth keeping some trace of the base name (`NodegroupCfg` or `CNodegroup` or something) just so it's more obvious looking at the code that it's not a CDK object (which was why everything was imported with `config`--it was getting super confusing differentiating things like `EKS`-the-cfg-object from `eks`-the-cdk-module the way everything was originally imported). Also (super minor nit, sorry), I don't think there's anywhere that capitalizes the `G` in the cdk libs.

Reply:
Though apparently we do capitalize it in one of the scopes here: https://github.com/dominodatalab/cdk-cf-eks/blob/243b19410e79bb630cb11cc1a2358e88d441d16a/cdk/domino_cdk/provisioners/eks/eks_nodegroup.py#L134

Reply:
Lmk what you think of 9bd5fc5ceeb728053d2910a3c84ea25ecdb1cc59?

Reply:
WFM